Examples/AutomateTest/README.md

```diff
@@ -0,0 +1,12 @@
+The goal of this script is to automate the testing of react-native, react-native-code-push and their internal compatibility. Run **./run.sh** to create an app and test it. 
```

Comment:
Can we have this simply as a comment at the top of the script? I don't think we need a separate README